modules/cloud-auth/cloud-auth-grammar.ym

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2023 Attila Szakacs
+ * Copyright (c) 2025 Axoflow
+ * Copyright (c) 2025 Attila Szakacs <attila.szakacs@axoflow.com>
```

Comment:
`2023-2025` here and the other files too (would be important to keep the original year too).